pkg/license/license.go

```diff
@@ -58,6 +60,9 @@ func getLicenseFromAPI(url string, licenseID string) (*LicenseData, error) {
 
 	req.SetBasicAuth(licenseID, licenseID)
 
+	instanceData := report.GetInstanceData(store.GetStore())
```

Comment:
It looks like there are cases where the store would not be initialized when calling `GetStore()` in this function. For example, this function is called by `GetLicenseByID` and `GetLatestLicense`, which are both called during the `bootstrap` function prior to initializing the in-memory store. In that case `GetStore` will return an empty struct and may result in some empty instance data headers.

This does not look like it would be an issue for the other two functions where we're making this change.

Reply:
Will this cause KOTS to panic?

Reply:
I don't believe the SDK will panic during bootstrapping since it looks like all tests are green. It may, however, cause some empty instance data headers to be sent to replicated-app since it looks like `GetStore()` returns an in-memory store with empty values when it hasn't been initialized. I don't know if that would actually cause any issues in reporting or not as it might depend on how replicated-app handles that.

Reply:
`replicated-app` is a bit defensive in this regard.  It has an conditional check along these lines:
```
 if (reportingInfo.instanceId && reportingInfo.clusterId) {
  ...
 }
 ```

Reply:
Gotcha, this is probably fine then. Something that we should fix eventually in the SDK, but I don't think that refactor needs to be part of this PR.

Reply:
@cbodonnell  I am going to go ahead and merge.